test cases/common/218 link custom_i multiple from multiple/generate_stlibs.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import shutil, sys, subprocess, argparse, pathlib
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--private-dir', required=True)
+parser.add_argument('-o', nargs='+', required=True)
+parser.add_argument('cmparr', nargs='+')
+
+contents = ['''#include<stdio.h>
+
+void flob_1() {
+    printf("Now flobbing #1.\\n");
+}
+''', 
+'''#include<stdio.h>
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5161#issue-4009488))

<sub>posted by Sider</sub>